### DISCUSSION TITLE
Official Documentation Auto Scroll Issue

### DISCUSSION THREAD

**Discussion by Leviash** [vuejs/pinia#2305] [Help and Questions] [answered]
When using 360 Extreme Browser to visit https://pinia.web3doc.top/, the page automatically scrolls down twice every approximately 500ms after scrolling down about 300px.

Just like this：
https://gitee.com/leviash/pics/raw/master/pics/document%20bug.gif

**Comment by posva** [comment, marked as answer]
The docs are at https://pinia.vuejs.org/
